### PR TITLE
Create "SIG PM representatives.md"

### PR DESCRIPTION
**Pull request by idvoretskyi**

### CODE REVIEW

**Comment by aveshagarwal** [comment]
sorry was looking something and by mistake removed the cncf cla labe;. reapplied.

File: sig-pm/SIG PM representatives.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+|  AWS | Justin Santa Barbara | justinsb |  |  |
+|  CLI | Joe Beda | jbeda | joe@bedafamily.com |  |
+|  Cluster Lifecycle | Dan Gillespie |  |  |  |
+|  Cluster Ops | Jason Singer DuMars | jdumars | jdumars@deis.com | Rob Hirschfeld |
+|  Contrib Experience |  |  |  |  |
+|  Docs |  |  |  |  |
+|  Federation | David Aronchick |  | aronchick@google.com |  |
+|  Instrumentation | Piotr Szczesniak | piosz | pszczesniak@google.com | Patrick Christopher (pat.christopher@gmail.com) |
+|  Network |  |  |  |  |
+|  Node | Caleb Miles | calebamiles | caleb.miles@coreos.com |  |
```
Comment (resouer) [inline sig-pm/SIG PM representatives.md]: Mind add `PengfeiNi | feiskyer | feiskyer@hyper.sh` for CRI (seems no this sig?) or Node here?

Comment (idvoretskyi) [inline sig-pm/SIG PM representatives.md]: cc @calebamiles as a SIG-Node representative

**Comment by idvoretskyi** [comment]
Merging with the latest updates; self-LGTM.

**Pull request merged**